backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/KeyValueHandler.java

```diff
@@ -135,7 +136,13 @@ public KeyValueHandler(Executor executor, FutureJdbi jdbi, String fieldType, Str
                 // Check for conflicts
                 jdbi.useHandle(
                     handle -> {
+                      Set<String> keySet = new HashSet<>();
                       for (final var kv : kvs) {
+                        if (keySet.contains(kv.getKey())) {
+                          throw new InvalidArgumentException(
+                              "Multiple Key " + kv.getKey() + " found in " + getTableName());
+                        }
+                        keySet.add(kv.getKey());
```

Comment:
If keyvalue like metrics, hyperparameters etc. list with same key multiple times then we have logged it without checking uniqueness for that list itself which was a bug so i have fixed it.